docs/netdata-cloud/authentication-and-authorization/role-based-access-model.md

```diff
@@ -130,6 +130,13 @@ In more detail, you can find on the following tables which functionalities are a
 | Manage payment methods     | :heavy_check_mark: |      -      |         -          |      -       | :heavy_check_mark: |     -      |                                                                 |
 | Update billing email       | :heavy_check_mark: |      -      |         -          |      -       | :heavy_check_mark: |     -      |                                                                 |
 
+### Dynamic Configuration Manager
+
+| **Functionality**     |     **Admin**      |    **Manager**     | **Troubleshooter** | **Observer** | **Billing** | **Member** | Notes |
+|:----------------------|:------------------:|:------------------:|:------------------:|:------------:|:-----------:|:----------:|:------|
+| Access and manage configuration items | :heavy_check_mark: | :heavy_check_mark: |         -          |      -       |      -      |     -      |       |
```

Comment:
Looking at this table I interpret it as only Admin and Manager roles can **view** and **edit** dyncfg configurations.


Should we split it into something like:

- List all (see all configurable items)
- Enable/Disable
- Restart
- Add
- Update
- Remove
- Test
- View
- View file format

I am not certain about "View file format" (the `useconfig` action). Alternatives:

- Format view
- Export

cc @shyamvalsan

Reply:
It is also important to note that all actions other than “List All” require a paid subscription.